tests/bdd/features/rebuild/test_log_based_rebuild.py

```diff
@@ -148,31 +151,26 @@ def a_volume_with_three_replicas_filled_with_user_data(disks):
     volume = ApiClient.volumes_api().put_volume_target(
         VOLUME_UUID,
         publish_volume_body=PublishVolumeBody(
-            {}, Protocol("nvmf"), node=NODE_1_NAME, frontend_node=""
+            {}, Protocol("nvmf"), node=NODE_1_NAME, frontend_node="app-node-1"
         ),
     )
 
     # Now the volume has been created, create an additional pool.
     ApiClient.pools_api().put_node_pool(
         NODE_4_NAME, POOL_4_UUID, CreatePoolBody([f"aio://{disks[3]}"])
     )
-
-    # Launch fio in background and let it always run along with the test.
-    """
-    uri = urlparse(volume["state"]["target"]["deviceUri"])
-    fio = Fio("job", "randrw", uri=uri, runtime=FIO_RUNTIME).build_for_userspace()
+    # Launch fio in background and let it run along with the test.
+    uri = volume["state"]["target"]["deviceUri"]
+    disk = nvme_connect(uri)
+    fio = Fio(name="job", rw="randwrite", device=disk, runtime=FIO_RUN)
+    pytest.fio = fio.open()
```

Comment:
I think you can just use fio = ...

Reply:
Moving this to separate def as i need to nvme_disconnect here